renew_certificate.sh

```diff
@@ -43,14 +43,17 @@ echo "Stopping Qthttpd hogging port 80.."
 
 /etc/init.d/Qthttpd.sh stop
 
-lsof -i tcp:80 -a -c python -t | xargs -r -I {} sh -c 'echo "Killing old python process {} hogging port 80" && kill {} && sleep 1'
+for PID in $(lsof -i tcp:80 -a -c python -t)
+do
+    echo "Killing old python process $PID hogging port 80" && kill $PID && sleep 1
```

Comment:
Can you split this up into one command per line?

Reply:
Yes sure, done.